### PR TITLE
Release 1.13.0

### PR DESCRIPTION
**Pull request by mwithi**
Preparing release 1.13.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ or any similar SQL database (e.g. MariaDB).
 
 ### Launch within IDE
 
-Be sure to have configured properly the classpath for the project (see [7 Installing Open Hospital 1.11.0 in Eclipse EE](https://github.com/informatici/openhospital-doc/blob/develop/doc_admin/AdminManual.adoc#7-installing-open-hospital-1-11-0-in-eclipse-ee))
+Be sure to have configured properly the classpath for the project (see [5 Installing Open Hospital 1.11.0 in Eclipse EE](https://github.com/informatici/openhospital-doc/blob/develop/doc_admin/AdminManual.adoc#5-installing-open-hospital-1-12-0-in-eclipse-ee))
```
Comment (dbmalkovsky) [inline README.md:28]: I think that this needs updating to be `1.13.0` (the title and the url).   Please see my suggested updates in the Admin doc.

File: oh.sh
Comment (dbmalkovsky) [inline oh.sh:1]: Line 45 has `OH_PATH=/usr/local/OpenHospital/oh-1.12` shouldn't it be `OH_PATH=/usr/local/OpenHospital/oh-1.13`?

Line 522 has `Version=1.12.0` shouldn't it `Version=1.13.0`?

Line 526 has `Comment=Open Hospital 1.12 shortcut` shouldn't it be `Comment=Open Hospital 1.13 shortcut`?

File: oh.ps1
Comment (dbmalkovsky) [inline oh.ps1:1]: Line 101 has `# OH_PATH="c:\Users\OH\OpenHospital\oh-1.12"` shouldn't it be `# OH_PATH="c:\Users\OH\OpenHospital\oh-1.13"`?

File: README.md (hunk 1 above)
Comment (mwithi) [reply README.md:28]: Done. Thank you

File: oh.sh
Comment (mwithi) [reply oh.sh:1]: yes, done. thank you!

File: oh.ps1
Comment (mwithi) [reply oh.ps1:1]: Done. Thank you

**Review by dbmalkovsky — APPROVED** [review]
LGTM

**Pull request merged**